core/src/main/kotlin/fr/sncf/osrd/api/stdcm/STDCMRequest.kt

```diff
@@ -57,6 +57,7 @@ class STDCMRequest(
     @Json(name = "temporary_speed_limits")
     val temporarySpeedLimits: Collection<STDCMTemporarySpeedLimit>,
     @Json(name = "work_schedules") val workSchedules: Collection<WorkSchedule> = listOf(),
+    val zones: Set<String>?,
```

Comment:
I'd still confuse that with detection zones with no context here, could it be renamed into something like `allowedTrackSections`? 

You'd probably prefer to avoid to propagate that change too far though (it may be a bit late for that). But we use the "other kind" of zone IDs in the rest of the editoast <=> core API, so it would be error prone here. It doesn't necessarily have to be changed further than that API call. 

The point where we change naming from "zone" to "allowed tracks" should be commented as well (wherever you decide to place it).